modules/medianetAnalyticsAdapter.js

```diff
@@ -382,6 +384,64 @@ class Auction {
   }
 }
 
+/**
+ * Here the idea is
+ * find all network entries via performance.getEntriesByType()
+ * filter it by video cache key in the url
+ * and exclude the ad server urls so that we dont match twice
+ * eg:
+ * dfp ads call: https://securepubads.g.doubleclick.net/gampad/ads?...hb_cache_id%3D55e85cd3-6ea4-4469-b890-84241816b131%26...
+ * prebid cache url: https://prebid.adnxs.com/pbc/v1/cache?uuid=55e85cd3-6ea4-4469-b890-84241816b131
+ *
+ * if the entry exists, emit the BID_WON
```

Comment:
I like the idea and it is a quick fix.
But i do not see that event BID_WON is triggered.
Can you please move this logic to core where it will be optionally enabled by a publisher using setConfig, 
Example: pbjs.setConfig({enableInstreamTracking: true});
This will help other analytics adapters to utilize it without knowing the actual implementation.